### PR TITLE
RemoteScheduler: use urljoin on base URL & suffix to form a full URL

### PR DESCRIPTION
**Pull request by paxan**

### CODE REVIEW

File: luigi/rpc.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import logging
 import socket
 import time
+from urlparse import urljoin
```
Comment (graingert) [inline luigi/rpc.py]: Python 3?

Comment (Tarrasch) [inline luigi/rpc.py]: Yea, you can iterate locally to catch these errors early @paxan, see [CONTRIBUTING](https://github.com/spotify/luigi/blob/master/CONTRIBUTING.rst) file. :)

Comment (paxan) [inline luigi/rpc.py]: My bad. I iterated locally only with py27 :(

**Comment by Tarrasch** [comment]
I usually wants commits to be squashed, but I might make an exception here as I suppose this is pretty urgent.

Thanks. :)

**Comment by paxan** [comment]
i will squash!

**Comment by Tarrasch** [comment]
Awesome, thanks for the quick fix @paxan, I hope this didn't have drastic consequences for your or anybody else production environments!

**Comment by paxan** [comment]
:+1:  (i've worked around it by ensuring scheduler_url arg has no trailing `/`)

**Comment by Tarrasch** [comment]
But after this patch it works regardless, right?

**Comment by geowurster** [comment]
I'll check master against my setup later today.

**Comment by geowurster** [comment]
@Tarrasch @paxan @graingert Looks like this is functioning correctly for me.  Thanks :+1:

**Comment by paxan** [comment]
@tarrasch right.

**Pull request merged**